web-client/src/views/CreateOrder/TextEditor.tsx

```diff
@@ -19,13 +21,40 @@ const ReactQuill = React.lazy(async () => {
   return { default: reactQuill };
 });
 
+// Quill does two things to render indentations:
+// 1) Adds padding-left so that the richText (and therefore the pdf) has indentations.
+// 2) Adds custom ql-indent-{level} classes to display the indentation in the Quill editor itself.
+// As part of the richText, 1 is automatically preserved across save and reload.
+// As part of the client-side Quill text rendering, 2 is *not* preserved across save and reload,
+// so even though the indentations are "there" in the richText, the user won't see them in the editor.
+// We call addQuillIndentationClasses on loading the text into the editor in order to correct that.
+const addQuillIndentationClasses = (html: string) => {
+  const dom = parseDocument(html);
+  // Traverse the DOM and add the appropriate ql-indent-{level} class corresponding to any elements with padding-left
+  DomUtils.findAll(elem => {
+    if (elem.name === 'p' && elem.attribs && elem.attribs.style) {
+      const { style } = elem.attribs;
+      const paddingLeftMatch = style.match(/padding-left:([0-9]+)em/);
+      if (paddingLeftMatch) {
+        const paddingLeftValue = parseInt(paddingLeftMatch[1]);
+        const indentLevel = paddingLeftValue / 3;
+        elem.attribs.class = `ql-indent-${indentLevel}`;
+      }
+    }
+    return true;
+  }, dom.children);
+
+  return renderHtml(dom);
+};
+
 export const TextEditor = ({
   defaultValue,
   editorDelta,
   updateFormValueSequence,
   updateScreenMetadataSequence,
 }) => {
   const quillEscapeRef = useRef(null);
+  defaultValue = addQuillIndentationClasses(defaultValue);
```

Comment:
Instead of reassigning the `defaultValue` that is passed in as an argument, could we instead make a new variable with a descriptive name and then use that new variable in the JSX?